integr8ly/keycloak-operator
# Intly 741

## Verification
- Create a namespace, e.g. `keycloak`
- Deploy this operator to that namespace
- Create the keycloak resource
- Create the keycloak realm resource (without createOnly)
- Wait for everything to come up
- Add a role to a users client roles in the realm
- Check the role is added in the rhsso ui
- Remove a role from a users client roles in the realm
- Check the role is removed in the rhsso ui
